server/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -635,11 +635,38 @@ public CommitStats commitStats() {
     }
 
     /**
-     * The sequence number service for this engine.
+     * @return the local checkpoint for this Engine
+     */
+    public abstract long getLocalCheckpoint();
+
+    /**
+     * @return the maximum sequence number from the local checkpoint tracker
+     */
+    public abstract long getMaxSeqNo();
```

Comment:
maybe we can remove this? it's only used in one place where I think we can just use getSeqNoStats?

Reply:
Certainly, I've done that, passing in -1 for the globalCheckpoint